src/test/java/com/wikia/webdriver/elements/oasis/pages/InfoboxBuilderPage.java

```diff
@@ -212,15 +280,23 @@ public InfoboxBuilderPage verifyDefaultTemplateStructure() {
     return this;
   }
 
-  public InfoboxBuilderPage verifySelectedComponentBorderStyle(int index) {
+  public InfoboxBuilderPage changeHeaderCollapsibilityState(int index) {
     JavascriptExecutor js = (JavascriptExecutor)driver;
-    WebElement selectedComponent = component.get(index);
-    wait.forElementVisible(selectedComponent);
-    selectedComponent.click();
-    String script = "return window.getComputedStyle("
-                    + "document.querySelector('.active'),':before').getPropertyValue('Border')";
-    String borderValues = js.executeScript(script).toString();
-    Assertion.assertEquals(borderValues, "1px solid rgb(26, 94, 184)");
+    headers.get(index).click();
+    wait.forElementVisible(collapsibilityCheckbox);
+    collapsibilityCheckbox.click();
+
+    String script = "return window.getComputedStyle(document"
+                    + ".querySelector('.pi-header'),':after').content";
+    String chevronContent = js.executeScript(script).toString();
+
+    if (collapsibilityCheckbox.isSelected()) {
+      Assertion.assertTrue(!chevronContent.isEmpty());
+    }
+    else
+    {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move this left curly brace to the end of previous line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ALeftCurlyBraceEndLineCheck)